### PR TITLE
RDM-4473 Add appInsights to case-management-web

### PR DESCRIPTION
**Pull request by tzarsmango**
https://tools.hmcts.net/jira/browse/RDM-4473

Added config changes for appInsights

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -1,3 +1,19 @@
 provider "azurerm" {
   features {}
 }
+
+data "azurerm_key_vault" "ccd_shared_key_vault" {
+  name                = "${var.raw_product}-${var.env}"
+  resource_group_name = "${var.raw_product}-shared-${var.env}"
+}
+
+data "azurerm_key_vault_secret" "appinsights_instrumentationkey" {
```
Comment (smathangi) [inline infrastructure/main.tf:10]: I don't think you nee anything here

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -14,6 +14,7 @@ def secrets = [
     'ccd-${env}': [
         secret('ccd-caseworker-autotest-fe-email', 'CCD_CASEWORKER_AUTOTEST_FE_EMAIL'),
         secret('ccd-caseworker-autotest-fe-password', 'CCD_CASEWORKER_AUTOTEST_FE_PASSWORD'),
+        secret('AppInsightsInstrumentationKey', 'APPINSIGHTS_INSTRUMENTATIONKEY'),
```
Comment (smathangi) [inline Jenkinsfile_CNP:17]: not required

File: infrastructure/main.tf (hunk 1 above)
Comment (rajaram-nagarajan) [reply infrastructure/main.tf]: Yes, removed.

File: Jenkinsfile_CNP (hunk 2 above)
Comment (rajaram-nagarajan) [reply Jenkinsfile_CNP]: Yes, removed

**Comment by rajaram-nagarajan** [comment]
Stale and moved the changes to https://github.com/hmcts/ccd-case-management-web/pull/1006

**Pull request closed without merging**